web/index.html

```diff
@@ -58,110 +52,144 @@
     <div class="container-fluid p-0">
 
         <!-- Remote Engineering Section-->
-        <div id="section-1" class="full-screen intro-remote-eng row justify-content-center px-5 d-flex align-items-center">
+        <div id="intro" class="row justify-content-center px-5 d-flex align-items-center full-screen intro-remote-eng ">
 
             <!-- xs breakpoint -->
-            <div class="col-xs-12 d-block d-sm-none">
-                <div>
-                    <h1 style="font-size:31px" class="re_text_animation_2" data-text="Remote&nbsp;Engineering">Remote  Engineering</h1>
+            <div class="col-xs-12  d-sm-none">
+                <div class="">
+                    <h1 class="intro-section-xs-breakpoint re_text_animation_sm_remote" data-text="Remote">Remote</h1>
+                    <h1 class="intro-section-xs-breakpoint re_text_animation_sm_engg" data-text="Engineering">Engineering</h1>
                     <p>Description on Remote Engineering</p>
                 </div>
             </div>
-            <div class="col-xs-12 d-block d-sm-none">
-                <img src="./assets/undraw_Freelancer_re_irh4.svg" alt="" height="300" width="500">
+            <div class="col-xs-12  d-sm-none">
+                <img class="img-fluid" src="/assets/intro-section.svg" alt="">
             </div>
 
             <!-- sm breakpoint -->
             <div class="col-sm-12 d-none d-sm-block d-md-none">
                 <div>
-                    <h1 style="font-size:35px" class="re_text_animation_2" data-text="Remote&nbsp;Engineering">Remote  Engineering</h1>
+                    <h1 class="intro-section-xs-breakpoint re_text_animation_sm_remote" data-text="Remote">Remote</h1>
+                    <h1 class="intro-section-xs-breakpoint re_text_animation_sm_engg" data-text="Engineering">Engineering</h1>
                     <p>Description on Remote Engineering</p>
                 </div>
             </div>
             <div class="col-sm-12 d-none d-sm-block d-md-none">
-                <img src="./assets/undraw_Freelancer_re_irh4.svg" alt="" height="300" width="500">
+                <img class="img-fluid" src="/assets/intro-section.svg" alt="" height="300" width="500">
```

Comment:
You might not need inline height and width if you are usign bootstrap responsive classes